java-client/session-dagger/src/test/java/io/deephaven/client/UpdateOrSelectSessionTest.java

```diff
@@ -51,8 +51,13 @@ public TableSpec apply(TableSpec spec, String[] formulas) {
             public TableSpec apply(TableSpec spec, String[] formulas) {
                 return spec.select(formulas);
             }
+        },
+        LAZY_UPDATE {
```

Comment:
This creates a conflict with your other PR, but you probably know that.

Reply:
Yep. I'm hoping to get #2635 merged in first, will fix conflicts here.